### DISCUSSION TITLE
the next button is not available

### DISCUSSION THREAD

**Discussion by midrisR** [gilbarbara/react-spotify-web-playback#81] [Q&A] [answered]
why isn't there a next button? how to add Next and previous buttons?
![image](https://user-images.githubusercontent.com/51366910/138571655-004b746e-1a5f-448f-b18a-5d092f67fe97.png)

**Comment by gilbarbara** [comment, marked as answer]
hey @midrisR 

You need to set an array of Spotify URIs in the `uris` prop
